### PR TITLE
Fix The Shaders And Includes

### PR DESCRIPTION
**Pull request by ckissane**
Did It!

### CODE REVIEW

File: src/main/java/Start.java [hunk 1]
```diff
@@ -8,8 +8,7 @@
 {
     public static void main(String[] args)
     {
-        System.out.println("Running Nova");
-        Main.main(concat(new String[] {"--version", "mcp", "--accessToken", "0", "--assetsDir", "assets", "--assetIndex", "1.10", "--userProperties", "{}"}, args));
+        Main.main(concat(new String[] {"--version", "mcp", "--accessToken", "0", "--assetsDir", "assets", "--assetIndex", "1.8", "--userProperties", "{}"}, args));
```
Comment (DethRaid) [inline src/main/java/Start.java]: Please undo this change. Asset index 1.10 is the correct one

**Review by DethRaid — COMMENTED** [review]
Did you run the nova-test executable to ensure that you didn't break anything?

**Review by DethRaid — COMMENTED** [review]
Why did you change the names of the shaders?

**Comment by ckissane** [comment]
nothing is broken

File: src/main/java/Start.java (hunk 1 above)
Comment (ckissane) [reply src/main/java/Start.java]: hmm I dont know how that happend
